android/java/org/chromium/chrome/browser/settings/BraveNewsPreferencesDataListener.java

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.settings;
+
+public interface BraveNewsPreferencesDataListener {
+    public void onChannelReceived();
```

Comment:
Modifier 'public' is redundant for interface methods

Reply:
Yeah, Nice catch! I'll remove it